test/cocotb/osdtestlib/eval_utility.py

```diff
@@ -54,9 +59,123 @@ def evaluate_response(self, dut, response, destination, source, reg_width,
         if result:
             # test was successful if none of the above mentioned errors occurred
             dut._log.info("Passed the test!")
+            dut._log.info("")
 
             # clear the buffer for the next read_register()
             del response[:]
 
-        for _ in range(1):
-            yield RisingEdge(dut.clk)
+
+        yield RisingEdge(dut.clk)
+
+
+    @cocotb.coroutine
+    def evaluate_dbpacket(self, dut, standard, id, trace_id, trace_value,
+                          response):
+        """Compares a received debug event packet with its expectation values
+           according to the trace ID and the trace value used to generate the
+           packet
+        Args:
+            dut                device under test.
+            standard           enum variable used to indicate if the packet
+                               should be evaluated according to old or the new
+                               format.
+            id                 ID of the debug module generating the debug event
+                               packet.
+            trace_id           ID of the trace event.
+            trace_value        Value of the trace event.
+            response           read debug event packet.
+        Returns:
+            -
+        """
+
+        # Currently the packets are still generated using the old format
+        # Changing the 'standard' variable will later on also enable the
+        # evaluation according to the new format
+        if (standard == Standard.old):
+
+            # Check the 1st flit
+            if (response[0] != 0x0000):
+
+                raise TestFailure("1st flit not ok!")
+
+            # Check the 2nd flit
+            elif ((((response[1] & 0xF000)>>12) != 0x0008)
```

Comment:
There's no need for an else after an early return (caused by a return or raise, as in this case).